daemon/src/config.rs

```diff
@@ -115,7 +115,12 @@ impl discovery::Discovery for StreamDiscovery {
                 for seed in seeds {
                     yield seed.into();
                 }
-                self.seeds_receiver.changed().await.unwrap();
+
+                match self.seeds_receiver.changed().await {
+                    Ok(_) => {},
+                    Err(_) => { break }
```

Comment:
Might be worth adding a log line here so we know the discovery is going away.

Reply:
Added the log line in https://github.com/radicle-dev/radicle-link/pull/688/commits/d1558357feb177b76ff06b883d545ea4f41708f7